From: Ganesh Goudar <ganeshgr@chelsio.com>
Date: Sat, 10 Mar 2018 17:34:50 +0530
Subject: cxgb4: do not display 50Gbps as unsupported speed
Patch-mainline: v4.17-rc1
Git-commit: 7cbe543cae9b57cf4f9e710d1cec8dc3dd3c6dda
References: bsc#1097585 bsc#1097586 bsc#1097587 bsc#1097588 bsc#1097583 bsc#1097584

50Gbps is a supported speed, Stop reporting it as
unsupported speed.

Signed-off-by: Ganesh Goudar <ganeshgr@chelsio.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -210,6 +210,9 @@ static void link_report(struct net_devic
 		case 40000:
 			s = "40Gbps";
 			break;
+		case 50000:
+			s = "50Gbps";
+			break;
 		case 100000:
 			s = "100Gbps";
 			break;
